### DISCUSSION TITLE
Allow disabling interactive behavior in code view

### DISCUSSION THREAD

**Discussion by jaskij** [community/community#58579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When browsing code in a repository (like in the screenshot below), I do not want a random single click in the code to open a side panel which does not benefit me.

I am not sure if this is a bug, or an intended feature, but when viewing a code file a single click in the code window will sometimes open the "Symbols" side panel. This is very distracting, and makes it harder to actually read the file, especially if the browser window is not full screen.

If it's a bug, it would be great to have it fixed, and if it's an intended feature I'd like an option to disable it.

My software:
- Arch Linux
- Gnome
- Firefox

FF plugins:
- uBlock origin
- Rust search extension
- Asciidoctor.js live preview

**Comment by Quuxplusone** [comment]
Yes please! I frequently screen-share and use the mouse to emphasize various parts of the code we're looking at. Now, every time I highlight a line of code, GitHub pops up a noisy "symbols pane" on the right-hand side of my screen — sometimes even hiding the line of code I'm trying to call attention to! And I have to move my mouse away from the relevant line of code in order to close the pop-up.

I want a way to browse the code without pop-ups.

**Comment by b2ag** [comment]
Added this to uBlock Origin
> github.com##.inner-panel-content-not-narrow.panel-content-narrow-styles

**Reply by kchung** [reply]
This didn't seem to work in my case, I used this one to disable the popover

```
github.com##.js-tagsearch-popover
```

**Comment by Ahmed7Version99** [comment]
you have to chnge your patch from 11.11 to another one you can do that by using versiouivate networkal

**Comment by kchung** [comment]
Please let us disable any of these "enhancements" and make it optional. Being forced to use a workflow that no one asked for or enabled is frustrating

**Comment by visortelle** [comment]
Dear GitHub, please let me disable it.
Sometimes I really need it, but it's a very rare use case.

**Comment by mattpolzin** [comment]
Same here. I just want to be able to highlight text; I edit files on my computer and have no interest in doing so inside GitHub via the web browser. On top of that, the interactive editor even shows when I don't have the _right_ to edit the file, which means sometimes I'll get an error that the file is read only in situation where I didn't want to use an editor to begin with!

**Comment by ajakaja** [comment]
+1 this is really disruptive

**Comment by StasD** [comment]
Yes, it's extremely annoying!
